datadog_checks_base/datadog_checks/base/checks/windows/perf_counters/refresh.py

```diff
@@ -0,0 +1,63 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import logging
+import threading
+import time
+from collections import defaultdict
+
+import pywintypes
+import win32pdh
+
+from ....utils.time import get_precise_time
+
+
+class WindowsPerformanceObjectRefresher(threading.Thread):
+    INTERVAL = 60
```

Comment:
You do not want to make it configurable and Andrew did?

Reply:
Also I think it worth to add maybe some random delta to stagger it bit because there is a chance that in some cases other PDH check (every 15 seconds) will always happen during enum and will be block every 60 seconds.

Reply:
> You do not want to make it configurable

No b/c it applies globally so nowhere to configure other than at the Agent level

> random delta

The 2 second sleep below will have the same effect

Reply:
I do not think that 2 seconds will help because it does not affect start of the check but on the other hand there is no hardcoded heartbeat on either checks.  In addition, Enum objects duration will be that natural randomness.